pkg/apis/core/v1/resource_tags.go

```diff
@@ -0,0 +1,74 @@
+package v1
+
+import (
+	"context"
+	"fmt"
+
+	"go.anx.io/go-anxcloud/pkg/api"
+	"go.anx.io/go-anxcloud/pkg/api/types"
+)
+
+// Adds tags to an object resource
```

Comment:
Your editor should complain about this:

```
$ golint ./pkg/apis/core/v1/
pkg/apis/core/v1/resource_tags.go:11:1: comment on exported function Tag should be of the form "Tag ..."
pkg/apis/core/v1/resource_tags.go:31:1: comment on exported function Untag should be of the form "Untag ..."
pkg/apis/core/v1/resource_tags.go:61:1: comment on exported function ListTags should be of the form "ListTags ..."
```

No idea why we don't have that in CI